Sources/NIOOpenSSL/SSLConnection.swift

```diff
@@ -49,22 +48,10 @@ internal final class SSLConnection {
         return false
     }
     
-    init? (_ ssl: UnsafeMutablePointer<SSL>, parentContext: SSLContext) {
+    init (_ ssl: UnsafeMutablePointer<SSL>, parentContext: SSLContext) {
```

Comment:
nit: extra space

Reply:
new suggestions: 
- either take ownership by upping or
- name `owned` like in the older PR